main.go

```diff
@@ -116,3 +120,15 @@ func (c *caPaths) Set(value string) error {
 	*c = append(*c, value)
 	return nil
 }
+
+func parseImageMirrors(mirrors string) map[string]string {
```

Comment:
This is better to use the `flag.Var` interface.
```go
type mirrorsMap map[string]string

func (m *mirrorsMap) String() string {
  return fmt.Sprintf("%v", *m)
}

func (m *mirrorsMap) Set(value string) error {
  result := strings.Split(value, "=")
  if len(result) != 2 {
    return errors.New("invalid format")
  }
  (*m)[result[0]] = result[1]
  return nil
}
```

Then you can use it like this:
```
--name docker.io=test.io
--name ghcr.io=test.io
```

Reply:
I totally agree